### DISCUSSION TITLE
Measuring the horizontal motion of the liquid centre of mass

### DISCUSSION THREAD

**Discussion by VVictorGit** [DualSPHysics/DualSPHysics#274] [DUALSPHYSICS_V5.2]
I would like to be able to track the liquid centre of mass in a tank partially filled with water, and then measure the horizontal displacement of that centre given a dynamic excitation in time of the tank with water inside. How would that be possible? 

I tried measure tool, I observed that one can define certain points and measure velocities, acceleration, etc. That includes the particle identity, but I cannot find a smart way to measure the displacement of a particular id particle. Perhaps I can identify the id particle of the liquid centre mass by defining the x,y,z point in a text file, but to be able to trace that through time, I couldn't do it. Is there an easy way to do it? I have seen that in the MeasureTool_help file one can create a predefined variable: "+/-XXX:       Variable defined by the user", but I am not really sure how to introduce that specifically to my problem within the DSPH formulation. 

I would appreciate an answer. Thank you!

**Comment by alexbexe** [comment]
What you try to do can not directly achieved with our Post-processing tools.. sorry for that.
However you can identify some particles in the initial centre of mass by their idp.
Use -onlyidp:XX,XX,XX in PartVTK to create files with the information of those particles and some resume CSV files should be created where you can see the position along time

Regards
Alex
